README.adoc

```diff
@@ -9,7 +9,7 @@
 :page-layout: guide
 :page-duration: 30 minutes
 :page-releasedate: 2017-09-19
-:page-description: Learn how to build a hypermedia-driven RESTful web service
+:page-description: Explore how to use Hypermedia as the Engine of Application State (HATEOAS) to drive your RESTful web service on Open Liberty.
```

Comment:
This seems a bit long, can it fit in the card?

Reply:
It is shorter than "Building fault-tolerant microservices with the @Fallback annotation", so it should.

Reply:
You can check it out if you have a local test environment setup, refer to: https://github.com/OpenLiberty/guides-common/wiki/Setting-up-your-local-test-website-and-Asciidoctor